react/FilterNavigator.js

```diff
@@ -51,18 +52,20 @@ const FilterNavigator = ({
     hints: { mobile },
   } = useRuntime()
 
-  const getSelectedFilters = () => {
-    const availableCategories = filter(prop('selected'), getCategories(tree))
+  const selectedFilters = useSelectedFilters(
+    useMemo(() => {
```

Comment:
correct me if I'm wrong, because I have not looked too deeply into this, but shouldn't hooks be used only on the top level of the component?

Reply:
yes! that's a good point but the `useMemo` is actually on the top level, and I wanted to "skip" one variable declaration (to avoid naming things). I think it kind looks weird at the first glance, because one wouldn't expect for it to be like this, but it's totally valid.